docs/src/prototypes/ComponentArchitecture/theme.ts

```diff
@@ -0,0 +1,23 @@
+export const customTheme: any = {
+  colors: {
+    background: 'white',
+    brand: 'blue',
+  },
+  fonts: {
```

Comment:
Animation will be from theme too? And it would be set here like fonts?

Reply:
Where would RTL and isInverted (like in the current scheme object in fabric) be set?

Reply:
Yes, all of those things should be here. Most of this is stubbed out and waiting for a final answer. 😄